flybywiresim/a32nx
# refactor: port SD to full react-based instrument

Fixes #[issue_no]

## Summary of Changes


- [x] point panel.cfg entry to template.html
- [x] port permanent data area
- [x] port logic (auto switching etc.)
~~-[ ] ensure accuracy (font sizes, layout...)~~ (later, when systems are in place)

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


Logic of the screens have not been touched by this PR so we don't expect any visual changes. However, here some points that can be checked.

1. Make sure all pages are selectable and show correct/meaningful data.
2. Test ECAM ALL button (pages should cycle in a one second interval)
3. Verify auto switching behaviour when no page is selected (non exhaustive list following)
   - Deselect any active ecam page when C&D and turn on the API -> APU page should appear
   - Start the engines -> ENG page should show up
   - While taxiing, move any control surface fully and verify the F/CTL page is showing up. When releasing controls, page should switch back after 20s
  - In cruise, CRZ page should be shown
4. General observation of any possible weird behaviour. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
